xml/System.Net/HttpStatusCode.xml

```diff
@@ -857,7 +858,7 @@ If the <xref:System.Net.HttpWebRequest.AllowAutoRedirect%2A?displayProperty=name
       </ReturnValue>
       <MemberValue>508</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Equivalent to HTTP status 508. <see cref="F:System.Net.HttpStatusCode.LoopDetected" /> indicates that the server terminated an operation because it encountered an infinite loop while processing a WebDAV request with "Depth: infinity". This status code is meant for backward compatibility with clients not aware of the 208 status code <see cref="F:System.Net.HttpStatusCode.AlreadyReported" /> appearing in multistatus response bodies.</summary>
```

Comment:
```suggestion
        <summary>Equivalent to HTTP status code 508. Indicates that the server terminated an operation because it encountered an infinite loop while processing a WebDAV request with "Depth: infinity". This status code is meant for backward compatibility with clients not aware of the 208 status code <see cref="F:System.Net.HttpStatusCode.AlreadyReported" /> appearing in multistatus response bodies.</summary>
```